demisto_sdk/commands/common/content/objects/pack_objects/__init__.py

```diff
@@ -52,7 +52,7 @@
 from .playbook.playbook import Playbook  # lgtm [py/polluting-import]
 from .pre_process_rule.pre_process_rule import \
     PreProcessRule  # lgtm [py/polluting-import]
-from .readme.readme import Readme  # lgtm [py/polluting-import]
+from .readme.readme import Readme, TextObject  # lgtm [py/polluting-import]
```

Comment:
Why do you export here the TextObject?
It would be best to export it from where you've created the object.
The README object isn't in charge of the TextObject, demisto_sdk/commands/common/content/objects/abstract_objects/text_object.py is. Remove this import (and the lgtm comment).